styles.css

```diff
@@ -0,0 +1,155 @@
+/*
+* Prefixed by https://autoprefixer.github.io
+* PostCSS: v8.4.14,
+* Autoprefixer: v10.4.7
+* Browsers: last 4 version
+*/
+
+body {
+    margin: 0 auto;
+    font-family: "Poppins", sans-serif;
+    background-image: url(./img/mobBg.png);
+}
+
+.mainMenu {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-pack: justify;
+        -ms-flex-pack: justify;
+            justify-content: space-between;
+    background-color: #FFFFFF;
+}
+
+.menuLogo {
+    color: #6070FF;
+    font-weight: 700;
+    font-size: 24px;
+    padding-left: 4%;
+    line-height: 6%;
+}
+
+.menuIcon {
+    position: relative;
+    top: 12px;
+    width: 22px;
+    right: 24px;
+    
+}
+
+.menuLvl {
+    display: block;
+    background-color: #6070FF;
+    width: 100%;
+    height: 2.5px;
+    margin: 4px;
+    border-radius: 2px;
+}
+
+.homeContent {
+    padding-left: 8%;
+    padding-right: 8%;
+}
+
+.homeContent h1 {
+    font-weight: 700;
+    font-size: 3rem;
+    line-height: 64px;
+    padding-top: 20%;
+}
+
+.contentPara {
+    margin-top: -10%;
+    font-size: 1rem;
+}
+
+.contentLink {
+    margin-top: -3%;
+    font-size: 1rem;
+    font-weight: 500;
+    color: #7F8CFF;
+}
+
+.homeContent li {
+    list-style-type: none;
+    margin-left: -15%;
+}
+
+.homeContent img {
+    height: 20px;
+    width: 20px;
+    margin-right: 4%;
+}
+
+
+@media screen and (min-width: 375px) {
+    body {
+        background-image: url(./img/deskBg.png);
+        background-repeat: no-repeat;
+        background-size: cover;
+    }
+    
+    .homeContent {
+        padding-left: 8%;
+        padding-right: 8%;
+    }
+
+    .homeContent h1 {
+        font-size: 4rem;
+        line-height: 75px;
+        padding-top: 10%;
+    }
+
+    .contentPara {
+        margin-top: -5%;
+        font-size: 1.5rem;
+    }
+
+    .contentLink {
+        margin-top: -2%;
+        font-size: 1.5rem;
+        font-weight: 500;
+        color: #7F8CFF;
+    }
+
+    .homeContent li {
+        list-style-type: none;
+        margin-left: -6.5%;
+    }
+}
+
+
+@media screen and (min-width: 768px) {
+    body {
+        background-image: url(./img/deskBg.png);
+        background-repeat: no-repeat;
+        background-size: cover;
+    }
+    
+    .homeContent {
+        padding-left: 16%;
+        padding-right: 16%;
+    }
+
+    .homeContent h1 {
+        font-size: 5rem;
+        line-height: 95px;
+        padding-top: 8%;
+    }
+
+    .contentPara {
+        margin-top: -3%;
+        font-size: 1.5rem;
+    }
+
+    .contentLink {
+        margin-top: -1%;
+        font-size: 1.5rem;
+        font-weight: 500;
+        color: #7F8CFF;
+    }
+
+    .homeContent li {
+        list-style-type: none;
+        margin-left: -3.5%;
```

Comment:
- [ ] NIce design so far :clap: :clap: , but you still to make it looks more professional, as required for this milestone of your project, Your website should look exactly like the design (e.g. font, colors, images, text, margins), and if we compare your design to the Figma one , they look somehow strange, kindly make sure that your design matches the Figma design as much as you can but you are almost there :wink: 

Your Design :point_down: 
![Screenshot from 2023-02-17 16-14-43](https://user-images.githubusercontent.com/88894525/219680501-cb9ef206-b914-4eab-bf60-fc5cc8587bc7.png)

Figma Design :point_down: 
![figmadesign](https://user-images.githubusercontent.com/88894525/219680582-2ebbf9a4-1b5b-4bba-83ef-cb3e2599ce54.png)

Reply:
Thanks for your kind feedback @Alfredbis29, I'll do my best! Have a great day!